### PR TITLE
feature: implement keyboard avoiding view

### PR DESCRIPTION
**Pull request by banklesss**
**Blocked until 4.23 is out**

------

Related to [YOMO-913](https://emurgo.atlassian.net/browse/YOMO-913)

Updated Screens:

- Confirm Voting Tx Form
- Delegation Confirmation Form
- Create/Restore Wallet Form
- Confirm TX Screens Form
- Start Multitoken Screen Form

[YOMO-913]: https://emurgo.atlassian.net/browse/YOMO-913?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: apps/wallet-mobile/src/Staking/DelegationConfirmation/DelegationConfirmation.tsx [hunk 1]
```diff
@@ -114,6 +114,8 @@ export const DelegationConfirmation = () => {
           chooseTransportOnConfirmation
         />
       </Actions>
+
+      {Platform.OS === 'ios' && <KeyboardSpacer />}
```
Comment (banklesss) [inline apps/wallet-mobile/src/Staking/DelegationConfirmation/DelegationConfirmation.tsx:118]: In this screen was impossible to use `KeyboardAvoidingView`. Explained [here](https://emurgo.atlassian.net/browse/YOMO-913?focusedCommentId=18844).

Finally I found an alternative to `KeyboardAvoidingView`, which is very buggy. 

This [task](https://emurgo.atlassian.net/browse/YOMO-978) has been created to try to find the reason why `KeyboadAvoidingView` can’t be implemented in this specific screen. If it’s not possible to implement it, i think its good idea to use `KeyboardSpacer` in all places where `KeyboadAvoidingView` is used to lift the bottom button.

File: apps/wallet-mobile/src/WalletInit/WalletForm.tsx [hunk 2]
```diff
@@ -118,6 +113,10 @@ export const WalletForm = ({onSubmit}: Props) => {
           testID="walletFormContinueButton"
         />
       </Actions>
+
+      {Platform.OS === 'ios' && <KeyboardSpacer />}
+
+      {Platform.OS === 'android' && <Spacer height={16} />}
```
Comment (banklesss) [inline apps/wallet-mobile/src/WalletInit/WalletForm.tsx:119]: the same thing happening here

https://github.com/Emurgo/yoroi/pull/2856/files#r1392753249

**Comment by stackchain** [comment]
blocking until 4.23 is out

**Comment by banklesss** [comment]
Changed to draft because `KeyboardSpacer ` will be implemented instead of `KeyboardAvoidingView`.

@stackchain

**Comment by banklesss** [comment]
Closing in favor of https://github.com/Emurgo/yoroi/pull/2936

**Pull request closed without merging**